backend/apps/monitoring/management/commands/verify_backup.py

```
@@ -0,0 +1,193 @@
+            local_file = download_dir / Path(backup_key).name
+            s3.download_file(s3_bucket, backup_key, str(local_file))
+
+            temp_db_name = f"backup_verify_{datetime.now().strftime('%Y%m%d%H%M%S')}"
+
+            env = os.environ.copy()
+            if db_password:
+                env["PGPASSWORD"] = str(db_password)
+
+            # Cleanup any existing temp DBs if possible (not done here for safety)
+            # Create DB
+            subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)